src/Compilers/CSharp/Test/Syntax/Parsing/RecordParsing.cs

```diff
@@ -325,45 +328,16 @@ public void RecordParsing07()
         {
             var text = "interface P(int x, int y);";
             var tree = ParseTree(text, options: TestOptions.Regular8);
-            tree.GetDiagnostics().Verify(
-                // (1,12): error CS1514: { expected
-                // interface P(int x, int y);
-                Diagnostic(ErrorCode.ERR_LbraceExpected, "(").WithLocation(1, 12),
-                // (1,12): error CS1513: } expected
-                // interface P(int x, int y);
-                Diagnostic(ErrorCode.ERR_RbraceExpected, "(").WithLocation(1, 12),
-                // (1,12): error CS8803: Top-level statements must precede namespace and type declarations.
-                // interface P(int x, int y);
-                Diagnostic(ErrorCode.ERR_TopLevelStatementAfterNamespaceOrType, "(int x, int y);").WithLocation(1, 12));
+            tree.GetDiagnostics().Verify();
```

Comment:
>tree.GetDiagnostics().Verify(); [](http://example.com/codeflow?start=12&length=31)

Please consider verifying the shape of the syntax tree since it looks like this may be the only test that tests an interface with a parameter list.
```
UsingNode(tree.GetRoot());

N(SyntaxKind.CompilationUnit);
...
```